### PR TITLE
[RFR] Improve the user experience

### PR DESCRIPTION
**Pull request by Luwangel**
Improve the user experience by changing the grid layout and the way to move pieces

- [x] Display the grid properly
- [x] Use the piece number instead of Top, Right Bottom and Left commands

### CODE REVIEW

File: src/game/game.py [hunk 1]
```diff
@@ -1,70 +1,67 @@
...
-    for x in range(0, size):
-        row = []
-        for y in range(0, size):
-            tile = x * size + y + 1
-            if tile > max_size:
-                tile = 0  # Corresponds to the empty box
-            row.append(tile)
-        grid.append(row)
-    return np.array(grid)
+    grid = np.arange(size ** 2).tolist()
```
Comment (Kmaschta) [inline src/game/game.py:6]: :+1:

File: src/game/game.py [hunk 2]
```diff
@@ -1,70 +1,67 @@
...
-    if not y == 0:
-        moves.append(Move.TOP)
-    if not x == len(grid) - 1:
-        moves.append(Move.RIGHT)
-    if not y == len(grid) - 1:
-        moves.append(Move.BOTTOM)
-    if not x == 0:
-        moves.append(Move.LEFT)
-    return moves
+    return tiles
```
Comment (Kmaschta) [inline src/game/game.py]: Please use .map(int)

File: src/game/test_game.py [hunk 3]
```diff
@@ -14,6 +13,24 @@ def test_build_grid(self):
             [13, 14, 15, 0]
         ])))
 
+    def test_find_tile(self):
```
Comment (Kmaschta) [inline src/game/test_game.py]: Can you find better test name that describes the tests?

**Review by Kmaschta — CHANGES_REQUESTED** [review]
Eventually, we need to test `index.py` code, or move its main function in `game`

**Comment by jpetitcolas** [comment]
Can you add an animated screenshot to better view what this PR does? 

You can use [Peek](https://github.com/phw/peek) for instance.

**Comment by Luwangel** [comment]
peek 16-10-2017 10-16

**Review by Kmaschta — APPROVED** [review]

**Pull request merged**